File: 1 - INICIANTE/python/1046.py
value = input().split(' ')
a, b = int(value[0]), int(value[1])
time = int()

if (a>b):
    time = (24-a)+b
elif (a==b):
    time = 24
else:
    time = b-a

print('O JOGO DUROU %d HORA(S)' %time)

File: 1 - INICIANTE/C++17/1046.cpp
#include <iostream>
#include <iomanip>
#include <cmath>
using namespace std;

int main() {
  int ho,hf,a,b;

  cin >> ho >> hf;

  a = (24 - ho) + hf;
  b = hf-ho; 

  if (ho >= hf) {
    cout << "O JOGO DUROU " << a << " HORA(S)" <<"\n";
  }
  else {
    cout << "O JOGO DUROU " << b << " HORA(S)" <<"\n";
  }
return 0;
}